sqlmesh/dbt/builtin.py

```diff
@@ -351,3 +351,17 @@ def create_builtin_globals(
 
 def create_builtin_filters() -> t.Dict[str, t.Callable]:
     return BUILTIN_FILTERS
+
+
+def _relation_info_to_relation(
+    relation_info: t.Dict[str, t.Any],
+    relation_type: t.Type[BaseRelation],
+    target_quote_policy: Policy,
+) -> BaseRelation:
+    merged_quote_policy_dict = {}
+    for k, v in (relation_info.pop("quote_policy", {})).items():
+        v = getattr(target_quote_policy, k, None) if v is None else v
+        if v is not None:
+            merged_quote_policy_dict[k] = v
+    quote_policy = Policy(**merged_quote_policy_dict)
```

Comment:
Could this be simplified to something like
{**target_quote_policy.dict(), **{k: v for k, v in relation_info.pop("quote_policy", {}) if v}}

Reply:
nice one 👍